sdk/src/main/java/io/opentelemetry/sdk/trace/export/SpanExporter.java

```diff
@@ -27,8 +27,6 @@
  * <p>To export data this MUST be register to the {@code TracerSdk} using a {@link
  * SimpleSpansProcessor} or a {@code BatchSampledSpansProcessor}.
  */
-// TODO: Change {@code BatchSampledSpansExporter} to {@link BatchSampledSpansExporter} when the
```

Comment:
Can we really remove this TODO?

Reply:
Actually, I think we can, as `BatchSampledSpansExporter` is actually (now) `BatchSpanProcessor`.

Reply:
yeah, I had no idea even what it meant.  :)